### DISCUSSION TITLE
Multiple Mailboxes

### DISCUSSION THREAD

**Discussion by pwahlmueller** [AdhocAdam/smletsexchangeconnector#461] [General]
It is possible to use multiple mailxboxes when forwarding all these mailboxes to one mailbox.

Is it somehow possible to directly use these mailboxes without forwarding?

**Comment by AdhocAdam** [comment]
If you mean directly attach to the additional mailboxes via EWS? Then by (current) design - no

**Reply by pwahlmueller** [reply]
That is what i meant. Thanks!

**Reply by AdhocAdam** [reply]
What led to this decision in the early days of the SMLets Exchange Connector came from a contributor:
> the HUGE performance impact caused by having to (i.e. actual business requirements) support a large number of distinct mailboxes put the Exchange Connector as the root cause of many issues we currently experience with SCSM.  It just doesn't scale--Microsoft verbally stated 15 as the maximum number anyone should attempt to run, and it is necessary for us to run more than 3x that number to meet our requirement.  If I replace all of those Exchange connectors with this single script running once every ~5 minutes, I can still monitor all of those mailboxes but without reducing my primary management server to paperweight status and breaking other workflows, etc.

As a result, this led to leveraging Exchange Mail Flow rules and a _single_ EWS connection to _single_ mailbox. Is there a reason you'd be looking to directly attach?

**Reply by pwahlmueller** [reply]
It's a requirement from security perspective - so not from me. I understand your point. I'll try to investigate that - may take some more time.

**Reply by pwahlmueller** [reply]
So the maximum number is on the Exchange Connector side in Service Manager. This is not an argument, if I run my connector as a powershell script copnnecting to 15 mailboxes.
